dino_runner/components/player_lives.py

```diff
@@ -22,4 +22,6 @@ def draw(self, SCREEN):
 
     def reduce_live(self):
         self.lives -= 1
-        print(self.lives)
+
+    def reset_lives(self):
+        self.lives = HEART_COUNT
```

Comment:
as we are using obstacle and powerup managers, I think this class should be also use that notation maybe it can be called life_manager ? or something like that?

Reply:
agree, it will be changed in the next PR